lib/Doctrine/Common/Cache/PhpFileCache.php

```diff
@@ -103,7 +103,7 @@ protected function doSave($id, $data, $lifeTime = 0)
         $code   = sprintf('<?php return %s;', $value);
 
         $ret = (file_put_contents($filename, $code) !== false);
-        chmod($filename, 0664);
+        @chmod($filename, 0664);
```

Comment:
I understand. Let's continue the discussion at https://github.com/doctrine/doctrine2/commit/caf30b889bb898620d843d1ec4940d01fa1d8877#commitcomment-13321422 (which is about the same issue and security problem) and come back later once we've got that sorted out.